core/src/test/java/org/elasticsearch/index/fielddata/AbstractFieldDataTestCase.java

```diff
@@ -94,8 +95,8 @@ protected boolean hasDocValues() {
         } else if (type.getType().equals("byte")) {
             fieldType = MapperBuilders.byteField(fieldName).docValues(docValues).fieldDataSettings(type.getSettings()).build(context).fieldType();
         } else if (type.getType().equals("geo_point")) {
-            // norelease update to .before(Version.V_2_2_0 once GeoPointFieldV2 is fully merged
-            if (indexService.getIndexSettings().getIndexVersionCreated().onOrBefore(Version.CURRENT)) {
+            if (type.getSettings().getAsVersion(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT).before(Version.V_2_2_0)
```

Comment:
Indeed! That other issue was setting version randomization to cover pre-2.0, when docvalues were not enabled. I pushed an update that purges this settings inconsistency.